dev-php/theseer-tokenizer/files/autoload.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/* Autoloader for dev-php/theseer-tokenizer */
+
+if (!class_exists('Fedora\\Autoloader\\Autoload', false)) {
+	require_once '/usr/share/php/Fedora/Autoloader/autoload.php';
```

Comment:
I think these would actually be better without the `/usr/share/php`, so instead...

```php
        require_once 'Fedora/Autoloader/autoload.php';
```

(thats lets us play games with the `include_dir` and override the default stuff with version-specific copies).